src/zenml/cli/server.py

```diff
@@ -242,6 +244,7 @@ def up(
                 f"'{server.status.url}'. You can connect to it using the "
```

Comment:
This log should change now to reflect that the browser is opening?

Reply:
IMO not necessary, people will notice :D

Reply:
Since we're using env variables, I'm putting a statement here to inform users about it. Is this necessary you think?